site/src/components/page-faq/question-section.tsx

```diff
@@ -0,0 +1,59 @@
+import { Component, h, Listen, Prop, State } from "@stencil/core";
+
+import { Fragment } from "../../util";
+
+/**
+* Component to take groups of faq questions and display them property
+* Also holds logic to toggle section open and closed
+*/
+@Component( {
+    tag: "question-section",
+    styleUrl: "page-faq.scss",
+    shadow: false,
+ })
+export class QuestionSection {
+
+    @Prop() public sectionTitle?: string;
+    @Prop() public questions?: { question: string, answer: () => string }[];
+
+    @State() private isOpen: boolean = false;
+
+    @Listen("click")
+    public handleClick() {
+      this.isOpen = !this.isOpen;
+    }
+
+    public render() {
+        const sectionTitle = this.sectionTitle || "";
+        const questions = this.questions || [];
+        const isOpen = this.isOpen;
+
+        return (
+            <Fragment>
+                <div>
+                    <hr />
+                    <header>
+                        <a onClick={this.handleClick}>
+                        <h3 class="section-title">
+                            { sectionTitle }
+                            <img
+                                alt={`toggle ${isOpen ? "open" : "close"}`}
+                                src={`/assets/images/${isOpen ? "icon_arrow-up.svg" : "icon_arrow-down.svg"}`}
+                                width="25" />
+                        </h3>
+                        </a>
+                    </header>
+                    <section>
+                    {isOpen && questions.map(({question, answer}) => (
```

Comment:
Could we animate this open and closed instead of rendering or not? e.g.
```scss
section { max-height: 0; overflow: hidden; transition: max-height 400 ease-in-out; &.open { max-height: 5000px; } }
```
And similarly for the arrow; maybe `transform: rotate(180deg);` and add a transition instead of swapping images?

Reply:
I am going to punt for the fast design follow, after this is merged and released.